src/util/parser-error-logger.js

```diff
@@ -1,14 +1,13 @@
 const {
   NODE_ENV,
-  REACT_APP_ENABLE_PARSER_DEBUGGING
+  REACT_APP_ENABLE_PARSER_DEBUGGING,
 } = process.env;
 
-let errorFn;
+const isDebuggingEnabled =
+  NODE_ENV === 'development' && REACT_APP_ENABLE_PARSER_DEBUGGING === 'true';
 
-if (NODE_ENV === 'development' && REACT_APP_ENABLE_PARSER_DEBUGGING === 'true') {
-  errorFn = (err) => { console.error(err); };
-} else {
-  errorFn = () => {};
-}
+const errorFn =
+  /* eslint-disable no-console */
+  isDebuggingEnabled ? (err) => { console.error(err); } : () => {};
```

Comment:
Since we use this syntax for all the other anonymous functions, wouldn't this be preferable?
`(err) => { console.error(err); }` => `err => console.error(err)`